e2e/utils/strategy/MyStrategyDriver.ts

```diff
@@ -41,6 +53,11 @@ export class MyStrategyDriver {
           marginalPrice: () => tooltip.getByTestId('marginal-price'),
         };
       },
+      clickManageEntry: async (id: ManageStrategyID) => {
+        await strategy.getByTestId('manage-strategy-btn').click();
+        await waitFor(this.page, 'manage-strategy-dropdown', 10_000);
+        await this.page.getByTestId(id).click();
+      },
```

Comment:
I just want to share how I have handle these cases in the other drivers : 
```suggestion
      manage: async () => {
        await strategy.getByTestId('manage-strategy-btn').click();
        const manager = await waitFor(this.page, 'manage-strategy-dropdown', 10_000);
        return {
          locator: manager,
          duplicate: () => manager.getByTestId('duplicate'),
          pause: () => manager.getByTestId('pause'),
          ...
        }
      },
```

Your code has the benefice of using types instead of typescript.
Let keep it like that, we can have a discussion at the end of the week to see how we want to harmonize.